### ISSUE TITLE
/account.get_wallets_and_user_wallets returns 500 when not providing account_id

### ISSUE THREAD

**Issue by jarindr** [omisego/ewallet#755]
This happen with account.get_wallets too.

**Comment by unnawut** [comment]
```
curl 'http://localhost:4000/api/admin/account.get_wallets' -H 'Origin: http://localhost:9000' -H 'Accept-Encoding: gzip, deflate, br' -H 'Accept-Language: en-US,en;q=0.9' -H 'Authorization: OMGAdmin dXNyXzAxZDJmMHBuZXphazk3ODl3c3oyNHo1ZHpqOkRYejFaMXA3NWdIRDQ1OTc5MkZ3WUJCTDBCaEk4ZlRGYm5sc2g2TXNqWUE=' -H 'Content-Type: application/json;charset=UTF-8' -H 'Accept: application/vnd.omisego.v1+json' -H 'Referer: http://localhost:9000/admin/accounts/acc_01d2f0s6wjj2bde509bf5jj0f0/detail' -H 'User-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/71.0.3578.98 Safari/537.36' -H 'Connection: keep-alive' --data-binary '{"per_page":10,"sort_by":"identifier","sort_dir":"desc","owned":true}' --compressed
```

**Comment by unnawut** [comment]
```
curl 'http://localhost:4000/api/admin/account.get_wallets' \
-H 'Authorization: OMGAdmin <truncated>' \
-H 'Content-Type: application/json;charset=UTF-8' \
-H 'Accept: application/vnd.omisego.v1+json'' \
-d '{"per_page":10,"sort_by":"identifier","sort_dir":"desc","owned":true}'
```

**Comment by unnawut** [comment]
Closes with #773.

Why isn't Github auto-closing issues anymore. 😞
